src/libhwloc_extensions.jl

```diff
@@ -0,0 +1,17 @@
+module LibHwlocExtensions
+
+using ..LibHwloc: libhwloc, Hwloc_jll
+
+using CEnum
+
+function hwloc_pci_class_string(class_id)
```

Comment:
Why is this in a new file/module and not just in `libhwloc.jl`?

Reply:
`libhwloc.jl` is machine generated (modulo a few edge cases that Clang.jl couldn't handle) by `generator.jl`. The intention is that if Hwloc ever chagned its ABI then `libhwloc.jl` would be regenerated, deleting any manual definitions. This definition is not part of Hwloc's `include/hwloc/*` header files, instead it's in `include/private/misc.h` (https://github.com/open-mpi/hwloc/blob/master/include/private/misc.h don't ask me why ...)

Anyway, this is to say that I want to preserve this manual addition to the ABI wrappers in the event of regenerating `libhwloc.jl`